src/main/java/ch/njol/skript/expressions/ExprExplodedBlocks.java

```diff
@@ -33,37 +40,135 @@
 import ch.njol.skript.doc.Since;
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
-import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.util.Kleenean;
 
 @Name("Exploded Blocks")
-@Description("Get all the blocks that were destroyed in an explode event")
-@Examples({"on explode:",
+@Description("Get all the blocks that were destroyed in an entity/block explode event (Can be set/removed from/deleted/added to)")
```

Comment:
```suggestion
@Description("Get all the blocks that were destroyed in an explode event")
```
The entity/block part makes it sound unusable in a generic explode event. Expressions currently don't have changer information in the docs, if it were to be added, it should be done to all of them